src/Framework/Framework/Compilation/ControlTree/ControlResolverBase.cs

```diff
@@ -56,8 +58,15 @@ public ControlResolverBase(DotvvmMarkupConfiguration configuration)
 			var searchKey = GetSearchKey(tagPrefix, tagName);
 			activationParameters = null;
 			var controlType = cachedTagMappings.GetOrAdd(searchKey, _ => FindControlType(tagPrefix, tagName));
-			if (controlType == null) return null;
-			return ResolveControl(controlType);
+			if (controlType is object) return ResolveControl(controlType);
+
+			if (tagPrefix == "js")
```

Comment:
Is it a good idea to hard-code the prefix here. If I was a library author and liked to build a library with JS controls (e. g. wrappers over existing control packs), there may be naming conflicts. 
I think we should allow registering custom prefixes for JS components.